src/Symfony/Component/Notifier/Bridge/Zulip/README.md

```diff
@@ -0,0 +1,12 @@
+Zulip Notifier
+===================
```

Comment:
Super minor comment: the `====` line should be as long as the previous header. Thanks.